include/MySQL_Thread.h

```diff
@@ -216,14 +216,21 @@ class MySQL_Thread
   ~MySQL_Thread();
   MySQL_Session * create_new_session_and_client_data_stream(int _fd);
   bool init();
-  void run();
+	void run___get_multiple_idle_connections(int& num_idles);
+	void run___cleanup_mirror_queue();
+  	void ProcessAllMyDS_BeforePoll();
```

Comment:
Mixed indentation in lines 221 and 222.

Reply:
Yes, we need to fix indentation in the original file, but I wanted to avoid this in this commit because not relevant . Can be done on a different cleanup